core/src/components/radio-group/radio-group.common.scss

```diff
@@ -1,38 +1,26 @@
 @import "../../themes/native/native.globals";
```

Comment:
This should be removed

Reply:
https://github.com/ionic-team/ionic-framework/pull/30271/commits/6ce549dd2f09e674d459e4a9a3ceb4c06915d887